### PR TITLE
Individual order details api added to smartapi sdk

### PR DESCRIPTION
**Pull request by Arvind-Acc**
1)Individual order details api added to smartapi sdk
2)now you can able to see the individual order details by passing your uniqueorderid

### CODE REVIEW

File: SmartApi/smartConnect.py [hunk 1]
```diff
@@ -433,6 +435,27 @@ def searchScrip(self, exchange, searchscrip):
...
             return searchScripResult
         
+    def ind_order_details(self, qParam):
+        url = self._rootUrl + self._routes["api.individual.order.details"] + qParam
+        headers = {
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+            "X-UserType": "USER",
+            "X-SourceID": "WEB",
+            "X-PrivateKey": self.api_key,
```
Comment (anurag-amx) [inline SmartApi/smartConnect.py:445]: These are common variables used across APIs. Use common method.

File: SmartApi/smartConnect.py [hunk 2]
```diff
@@ -433,6 +435,27 @@ def searchScrip(self, exchange, searchscrip):
...
+        }
+
+        if self.access_token:
+            headers["Authorization"] = "Bearer " + self.access_token
```
Comment (anurag-amx) [inline SmartApi/smartConnect.py:449]: This access_token processing is also same. use common method

File: SmartApi/smartConnect.py [hunk 3]
```diff
@@ -433,6 +435,27 @@ def searchScrip(self, exchange, searchscrip):
...
+        response = requests.get(url, headers=headers)
+
+        if response.status_code == 200:
+            data = json.loads(response.text)
```
Comment (anurag-amx) [inline SmartApi/smartConnect.py:453]: This is also same processing . Use common method.

File: SmartApi/smartConnect.py [hunk 4]
```diff
@@ -433,6 +435,27 @@ def searchScrip(self, exchange, searchscrip):
...
+            return data
+        else:
+            print("Error:", response.status_code)
+            return None
```
Comment (anurag-amx) [inline SmartApi/smartConnect.py:457]: Pass error response in case status !=200.

**Review by anurag-amx — APPROVED** [review]

File: SmartApi/smartConnect.py (hunk 2 above)
Comment (Arvind-Acc) [reply SmartApi/smartConnect.py:449]: created common method

File: SmartApi/smartConnect.py (hunk 3 above)
Comment (Arvind-Acc) [reply SmartApi/smartConnect.py:453]: created common method

File: SmartApi/smartConnect.py (hunk 1 above)
Comment (Arvind-Acc) [reply SmartApi/smartConnect.py:445]: created common method

File: SmartApi/smartConnect.py (hunk 4 above)
Comment (Arvind-Acc) [reply SmartApi/smartConnect.py:457]: created common method

**Review by anurag-amx — APPROVED** [review]

**Review by gulshan178 — APPROVED** [review]

**Pull request merged**